subprojects/core-api/src/main/java/org/gradle/api/provider/Property.java

```diff
@@ -45,6 +45,24 @@
      */
     void set(@Nullable T value);
 
+    /**
+     * @since 6.1
+     */
+    @Incubating
+    void setValue(T value);
+
+    /**
+     * @since 6.1
+     */
+    @Incubating
+    Provider<? extends T> getSource();
```

Comment:
This is required to make Kotlin happy. It does not consider `source` a property unless there is a getter method.